include/gz/math/Matrix3.hh

```diff
@@ -21,9 +21,8 @@
 #include <cstring>
 #include <utility>
 #include <gz/math/Helpers.hh>
-#include <gz/math/Vector3.hh>
-#include <gz/math/Quaternion.hh>
```

Comment:
Quaternion is used in this header, so I think we should keep this header. We can probably remove the forward declaration below actually.